3_javascript/javascript3/week2/script.js

```diff
@@ -0,0 +1,21 @@
+console.log("Let's start craziness")
+
+const convert = document.getElementById("convert");
```

Comment:
The API should populate all currency options instead of just EUR to DKK conversion. The code should be rewritten to comply with the homework.